### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by jainaman224**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -305,7 +305,7 @@ <h2 class="fh5co-sub-lead to-animate">100% Free Fully Responsive HTML5 Bootstrap
...
+                            <h3> Google Developer Groups (GDGs) are for developers who are interested in Google's developer technology. Everything from the Android, App Engine, and Google Chrome platforms, to product APIs. We at GDG-BVP, use Google Technologies, in-order to produce some great solutions and applications. GDG-BVP was started as an initiative to promote the use of Google &amp; other open Source technologies. We believe, we are a group of students who can code, design &amp; use technology to innovate. We love to share our knowledge and simultaneously promote the use of Google technologies.</h3>
```
Comment (poush) [inline index.html]: Can you explain what you did here? It looks exact same as before.

**Comment by poush** [comment]
@jainaman224 Please use descriptive message that what you updated in index.html

**Comment by jainaman224** [comment]
I have fixed typo in that line.
It should be We love to share our knowledge and simultaneously promoting the use of Google technologies. 
Instead of We love to share our knowledge and simultaneous promote the use of Google technologies.

**Comment by poush** [comment]
Great, Thanks. Also please use descriptive commit messages from next time.

**Pull request merged**